index.html

```diff
@@ -156,9 +156,10 @@
            leverages existing discovery mechanisms but does not directly
            expose metadata; they are simply used to discover Exploration
            services, which provide metadata but only after secure authentication
-           and authorization.  This document normatively defines two Exploration
-           services, one for WoT Thing self-description with a single 
-           WoT Thing Description and a searchable WoT Thing Description Directory
+           and authorization.  This document normatively defines three Exploration
+           services: for distributing a single WoT Thing Description from a regular
+           web service; for WoT Thing self-description providing a single 
+           WoT Thing Description; and a searchable WoT Thing Description Directory
```

Comment:
```suggestion
           and authorization.  This document normatively defines three Exploration
           services: for distributing a single WoT Thing Description from a regular
           web server; for WoT Thing self-description providing a single 
           WoT Thing Description; and a searchable WoT Thing Description Directory
```
Or "as a regular web service". I will use web server in the next suggestions.

Reply:
I did change "Thing Description Service" to "Thing Description Server" generally, but may have missed a few...